VideoLocker/res/values/errors.xml

```diff
@@ -63,6 +63,19 @@
     <!-- Alert dialog content shown when user attempts to view a video that hasn't been synced -->
     <string name="video_offline_message">Please select a video that you’ve downloaded onto your device.</string>
 
+    <!-- New version available -->
+    <!-- Snackbar notification shown when the current version of the app is unsupported -->
+    <string name="app_version_unsupported">Your version of the app is no longer supported. Upgrade to the latest version.</string>
+    <!-- Snackbar notification shown when the current version of the app is deprecated -->
+    <string name="app_version_deprecated">Your version of the app is no longer supported. Upgrade to the latest version by {last_supported_date} to continue accessing course content online.</string>
+    <!-- Snackbar notification shown when the current version of the app is outdated -->
+    <string name="app_version_outdated">A new version of the app is available. Upgrade to the latest version.</string>
+    <!-- Snackbar action button label for opening an app store to upgrade the app to the latest version -->
+    <string name="app_version_upgrade_button">Upgrade</string>
```

Comment:
@brianguertin pointed out that the more accurate term is "Update" to a version, rather than "Upgrade". If this button is used only for these types of version changes, can we change the button name here to "Update"? @1zaman @verdanmahmood @lizcohen FYI @mikekatz